modules/virt-customizing-storage-profile.adoc

```diff
@@ -67,3 +67,37 @@ status:
 <2> The `volumeMode` that you select.
 +
 After you save your changes, the selected values appear in the storage profile `status` element.
+
+== Setting a default cloning strategy using a storage profile
+
+You can use storage profiles to set a default cloning method for a storage class, creatng a _cloning strategy_. Setting cloning strategies  can be helpful, for example, if your storage vendor only supports certain cloning methods. It also allows you to select a method that limits resource usage or maximizes performance.
+
+Cloning strategies can be specified by setting the `cloneStrategy` attribute in a storage class to one of these values:
+
+* `snapshot` - The default cloning method, using smart cloning. Smart cloning quickly and efficiently creates a clone by cloning the data volume PVC to another PVC.
+* `copy` - Host-assisted cloning, used if your storage vendor does not support smart cloning. Host-assisted cloning is the least efficient method of cloning.
```

Comment:
"used if your storage vendor does not support csi clones or csi snapshots"?  This method uses a source Pod and a target Pod to copy data from the source volume to the target volume.

Reply:
Changed to: 

This method uses a source Pod and a target Pod to copy data from the source volume to the target volume. Host-assisted cloning is the least efficient method of cloning.